portfolio/src/main/webapp/style.css

```diff
@@ -20,24 +20,43 @@ ins {
     background-color: #d4fcbc; /*light-green*/
 }
 
-.highlight {
-    background-color: yellow;
-}
-
 input:focus {
     outline-width: 0; /*i don't want that ugly outline on any input */
 }
 .hidden {
     visibility: hidden;
 }
 
+.highlight {
+    background-color: yellow;
+    display: inline-block;
+    animation-name: hi-animation;
+    animation-iteration-count: 1;
+    animation-timing-function: ease-in-out;
+    animation-duration: 2s;
+}
+
+@keyframes hi-animation {
+    0% {
+        transform: scale(1);
+    }
+
+    50% {
+        transform: scale(1.5);
+    }
+    100% {
+        transform: scale(1);
+    }
+}
+
 .fade-in {
 	opacity: 1;
 	animation-name: fadeInOpacity;
 	animation-iteration-count: 1;
 	animation-timing-function: ease-in;
-	/* accounting for page loading time, if it takes more than 4 seconds to load no point in fading in anyways */
-    animation-duration: 4s; 
+    /* accounting for page loading time,
+     if it takes more than 4 seconds to load no point in fading in anyways */
+	animation-duration: 4s; 
```

Comment:
nit: i think indentation is off here?

Reply:
i'm still seeing the comment need two more spaces of indent to be inline with this block